filebeat/docs/configuring-howto.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[configuring-howto-filebeat]]
+== Configuring Filebeat
```

Comment:
Yah == is right because we are going to have top-level container for grouping topics related to configuration. You can see how it looks in the built files here: https://filebeatreference.firebaseapp.com/

The basic config steps are covered in the Getting Started.